tests/covalent_tests/workflow/electron_test.py

```diff
@@ -69,6 +74,41 @@ def workflow_2():
     return res_3
 
 
+def test_build_sublattice_graph():
+    """
+    Test building a sublattice graph
+    """
+
+    @ct.electron
+    def task(x):
+        return x
+
+    @ct.lattice
+    def workflow(x):
+        return task(x)
+
+    parent_metadata = {
+        "executor": "parent_executor",
+        "executor_data": {},
+        "workflow_executor": "my_postprocessor",
+        "workflow_executor_data": {},
+        "deps": {"bash": None, "pip": None},
+        "call_before": [],
+        "call_after": [],
+        "triggers": None,
+        "results_dir": None,
+    }
+
+    json_lattice = _build_sublattice_graph(workflow, json.dumps(parent_metadata), 1)
+    lattice = Lattice.deserialize_from_json(json_lattice)
+
+    assert list(lattice.transport_graph._graph.nodes) == list(range(3))
+    for k in lattice.metadata.keys():
+        # results_dir will be deprecated soon
+        if k != "results_dir":
```

Comment:
IIRC triggers are also not copied from parent lattice. Can you also check that by giving triggers a mock value and then comparing the sublattice and parent lattice's triggers for inequality?